### PR TITLE
Zotero updates

### PR DESCRIPTION
**Pull request by Robinlovelace**

### CODE REVIEW

File: geocompr.bib [hunk 1]
```diff
@@ -1,34 +1,37 @@
...
+  doi = {10.3133/70047422},
+  url = {https://doi.org/10.3133/70047422},
+  organization = {US Geological Survey}
 }
 
 @book{abelson_structure_1996,
   title = {Structure and Interpretation of Computer Programs},
   author = {Abelson, Harold and Sussman, Gerald Jay and Sussman, Julie},
-  year = {1996},
+  date = {1996},
```
Comment (Nowosad) [inline geocompr.bib:12]: @Robinlovelace -- I think this is a wrong export (I think that you used Better BibLatex, but should Better Bibtex, but I am not sure)

Comment (Robinlovelace) [reply geocompr.bib:12]: yeah maybe, I noticed that. Will try again.

Comment (Robinlovelace) [reply geocompr.bib:12]: Live update:

image

**Comment by Robinlovelace** [comment]
It's stuck trying to export: 

image

**Comment by Robinlovelace** [comment]
@Nowosad fixed with latest release: https://github.com/retorquere/zotero-better-bibtex/releases

Can you take a look and merge/test as appropriate?

File: geocompr.bib [hunk 2]
```diff
@@ -522,11 +541,11 @@ @article{conrad_system_2015
...
 
-@manual{cooley_sfheaders_2020,
-  type = {Manual},
-  title = {Sfheaders: {{Converts}} between r Objects and Simple Feature Objects},
+@book{cooley_sfheaders_2020,
+  title = {Sfheaders: {{Converts}} between \{\vphantom\}{{R}}\vphantom\{\} Objects and Simple Feature Objects},
   author = {Cooley, David},
-  year = {2020}
+  year = {2020},
+  publisher = {R package}
```
[the remark this answers is not in the document]
Comment (Nowosad) [reply geocompr.bib:548]: Yes, surely. Or even both (e.g, Lovelace R. (2024) somepackage. R package, CRAN)

**Review by Nowosad — APPROVED** [review]

**Pull request merged**